### PR TITLE
Remove unneeded argoproj crds from smaug.

### PR DESCRIPTION
**Pull request by HumairAK**
These are being managed by ODH operator:

```
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
  annotations:
    kfctl.kubeflow.io/kfdef-instance: opendatahub.opf-argo
  name: cronworkflows.argoproj.io

---
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
  annotations:
    kfctl.kubeflow.io/kfdef-instance: opendatahub.opf-argo
  name: workflows.argoproj.io
---
apiVersion: apiextensions.k8s.io/v1
kind: CustomResourceDefinition
metadata:
  annotations:
    kfctl.kubeflow.io/kfdef-instance: opendatahub.opf-argo
  name: workflowtemplates.argoproj.io
```

### CODE REVIEW

**Review by Gregory-Pereira — APPROVED** [review]
Understood, /Lgtm 👍 .

**Comment by harshad16** [comment]
/hold
why are these unneeded ?

**Comment by HumairAK** [comment]
@harshad16 I should have been clearer, they are unneeded in the `smaug` kustomization, they are being added by `odh` operator, see the post above, if you view these crd's you'll see odh labels, indicated they were added by odh.

**Comment by harshad16** [comment]
/unhold
lgtm then, I wasn't sure if the odh would allow it in all namespaces.

**Comment by HumairAK** [comment]
@harshad16 CRD's are cluster wide, we're not changing any rbac

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**
